src/tlo/methods/equipment.py

```diff
@@ -104,35 +104,25 @@ def availability(self, value: Literal["all", "default", "none"]):
         assert value in {"all", "none", "default"}, f"New availability value {value} not recognised."
         self._availability = value
 
-    def _calculate_equipment_availability_probabilities(self) -> pd.Series:
+    def _get_equipment_availability_probabilities(self) -> pd.Series:
         """
-        Compute the probabilities that each equipment item is available (at a given
+        Extract the probabilities that each equipment item is available (at a given
         facility), for use when the equipment availability is set to "default".
 
-        The probabilities computed in this method are constant throughout the simulation,
+        The probabilities extracted in this method are constant throughout the simulation,
         however they will not be used when the equipment availability is "all" or "none".
-        Computing them once and storing the result allows us to avoid repeating this
+        Extracting them once and storing the result allows us to avoid repeating this
         calculation if the equipment availability change event occurs during the simulation.
         """
-        # Create "full" dataset, where we force that there is probability of availability for every item_code at every
-        # observed facility
-        dat = pd.Series(
-            index=pd.MultiIndex.from_product(
+        dat = self.data_availability.set_index(
+            [self.data_availability["Facility_ID"].astype(int), self.data_availability["Item_Code"].astype(int)]
+        )["Pr_Available"]
```

Comment:
We had a conversation before about whether equipment should have a probability of availability with values between 0 to 1 or just have a binary value to indicate whether each equipment is available or not at each Facility_ID.  And I believe we decided to go with the latter. @tbhallett, should we update the equipment availability with a rule like this - 
`final_equipment_availability_export_full[final_equipment_availability_export_full < 0.5] = 0
final_equipment_availability_export_full[final_equipment_availability_export_full >= 0.5] = 1`

Note that this increases the average availability of equipment from ~60% to ~ 73%.

Reply:
Good question.

The implementation I've proposed (and which is in `master`) [see below] at the moment would need us to retain these probabilities. 

I think working on the availability of a piece of equipment probabilistically for the HSI_Event is a better way to represent that these data estimate are really telling us about the *proportion* of facilities in a particular level/district that have that piece of equipment.

https://github.com/UCL/TLOmodel/blob/67c80589a2e0f274ac12c87cd76876df3010d51d/src/tlo/methods/equipment.py#L13-L61